myapp/app/finders/tasks_finder.rb

```diff
@@ -0,0 +1,37 @@
+class TasksFinder
```

Comment:
In my opinion, better than Finders is places on services directory than here

Reply:
👌🏻